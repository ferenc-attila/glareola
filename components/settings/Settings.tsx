import {StyleSheet, Text, View} from "react-native";

export const Settings = () => {
    return (
        <View style={styles.textContainer}>
            <Text
                style={styles.title}
            >Settings</Text>
        </View>
    )
}

const styles = StyleSheet.create({
    textContainer: {
        flex: 1,
        backgroundColor: '#2e3a24',
        alignItems: 'center',
        justifyContent: 'center',
    },
    title: {
        color: '#fff',
        fontWeight: '900',
        fontSize: 22,
    }
});
