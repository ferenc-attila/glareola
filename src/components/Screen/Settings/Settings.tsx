import { StyleSheet, Text } from 'react-native';

import i18n from '../../../localization';
import { textStyles } from '../../../styles';

export const Settings = () => {
    return <Text style={styles.description}>{i18n.t('SCREEN_SETTINGS_DESCRIPTION')}</Text>;
};

const styles = StyleSheet.create({
    description: {
        ...textStyles.commonText,
    },
});
