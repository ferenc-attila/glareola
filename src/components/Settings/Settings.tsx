import {StyleSheet, Text, View} from "react-native";

export const Settings = () => {
    return (
        <View style={styles.textContainer}>
            <Text style={styles.title}>
                Settings
            </Text>
            <Text style={styles.description}>
                Coming soon...
            </Text>
        </View>
    )
}

const styles = StyleSheet.create({
    textContainer: {
        flex: 1,
        backgroundColor: '#2e3a24',
        alignItems: 'center',
        justifyContent: 'center',
    },
    title: {
        color: '#fff',
        fontWeight: '900',
        fontSize: 22,
        padding: 10,
    },
    description: {
        color: '#fff',
        fontWeight: '400',
        fontSize: 16,
    },
});
