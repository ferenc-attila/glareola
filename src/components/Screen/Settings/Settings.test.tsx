import { render } from '@testing-library/react-native/build/pure';

import { Settings } from './Settings';

describe('Settings page', () => {
    test('should render correctly', () => {
        const container = render(<Settings />);
        expect(container.toJSON()).toMatchSnapshot();
    });
});
