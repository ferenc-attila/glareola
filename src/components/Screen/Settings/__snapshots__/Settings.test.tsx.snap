// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings page should render correctly 1`] = `
<Text
  style={
    {
      "color": "#fff",
      "fontSize": 14,
      "fontWeight": "500",
    }
  }
>
  Coming soon...
</Text>
`;
